### DISCUSSION TITLE
Does RemixRun official suggests the best or the most suitable way to write css in Remix ?

### DISCUSSION THREAD

**Discussion by WreewanMorhee** [remix-run/remix#9363] [General]
Like title, what is the best way to write css in Remix Run ?
or  the best way RemixRun official suggests ?

Because I found some kind of way to write in doc like: regular, sass, tailwind, css-in-js, postCSS ... etc
But I want to know is there the best or the most suitable way to write css in remix

**Comment by kiliman** [comment]
I'm biased, but I believe Tailwind is the best way to write CSS, with or without Remix.

However, projects have their own preferences, so Remix tries to cater to their needs. With the new Vite compiler, Remix leaves all the heavy lifting to Vite. That's why the documentation links directly to the Vite documentation.
